rust/sealevel/programs/mailbox/Cargo.toml

```diff
@@ -22,11 +22,11 @@ spl-noop.workspace = true
 getrandom = { workspace = true, features = ["custom"] }
 
 proc-macro-crate = "~1.2.1" # TODO: remove this dependency once solana supports rust >=1.64
-blake3 = "=1.4.0" # TODO: update once solana supports rust >= 1.66
+blake3 = "=1.4.0"           # TODO: update once solana supports rust >= 1.66
```

Comment:
ditto

Reply:
This is automatic for me to line up the TOML comments using Even Better TOML, should I override this?

Reply:
Discussed with @daniel-savu to leave it as a part of TOML formatting